client/my-sites/theme/live-preview-button/index.jsx

```diff
@@ -0,0 +1,129 @@
+import { Button } from '@automattic/components';
+
+/**
+ * Hardcoded list of themes that are not compatible with Block Theme Previews.
+ * This list should be removed once they are retired.
+ *
+ * @see pekYwv-284-p2
+ */
+const NOT_COMPATIBLE_THEMES = [
+	'premium/alonso',
+	'premium/baxter',
+	'premium/byrne',
+	'premium/kerr',
+	'premium/munchies',
+	'premium/payton',
+	'premium/skivers',
+	'premium/thriving-artist',
+	'pub/ames',
+	'pub/antonia',
+	'pub/appleton',
+	'pub/arbutus',
+	'pub/attar',
+	'pub/barnett',
+	'pub/bennett',
+	'pub/calvin',
+	'pub/calyx',
+	'pub/course',
+	'pub/dorna',
+	'pub/farrow',
+	'pub/geologist-blue',
+	'pub/geologist-cream',
+	'pub/geologist-slate',
+	'pub/geologist-yellow',
+	'pub/hari',
+	'pub/heiwa',
+	'pub/jackson',
+	'pub/kingsley',
+	'pub/marl',
+	'pub/meraki',
+	'pub/quadrat-black',
+	'pub/quadrat-green',
+	'pub/quadrat-red',
+	'pub/quadrat-white',
+	'pub/quadrat-yellow',
+	'pub/quadrat',
+	'pub/russell',
+	'pub/seedlet-blocks',
+	'pub/winkel',
+];
+
+const isNotCompatibleThemes = ( stylesheet ) => {
+	return NOT_COMPATIBLE_THEMES.includes( stylesheet );
+};
+
+/**
+ * Live Preview leveraging Gutenberg's Block Theme Previews
+ *
+ * @see pbxlJb-3Uv-p2
+ */
+export const LivePreviewButton = ( {
+	isActive,
+	isAtomic,
+	isExternallyManagedTheme,
+	isFullSiteEditingTheme,
+	isSimple,
+	isThemeInstalledOnAtomicSite,
+	isWporg,
+	siteSlug,
+	stylesheet,
+	themeType,
+	themeId,
+	translate,
+} ) => {
+	// A user doesn't want to preview the active theme.
+	if ( isActive ) {
+		return null;
+	}
+
+	// A theme should be FullSiteEditing compatible to use Block Theme Previews.
+	if ( ! isFullSiteEditingTheme ) {
```

Comment:
I did not use `showTryAndCustomize` here as `showTryAndCustomize = true` does not mean it's not FullSiteEditing compatible. I thought it'd be nice to show the Live Preview button anytime in cases Block Theme Previews is available.